akka-docs/src/test/scala/docs/stream/FlowStreamRefsDocSpec.scala

```diff
@@ -76,13 +78,13 @@ class FlowStreamRefsDocSpec extends AkkaSpec with CompileOnlySpec {
           val sink: Sink[String, NotUsed] = logsSinkFor(nodeId)
 
           // materialize the SinkRef (the remote is like a source of data for us):
-          val ref: SinkRef[String] = Source.sinkRef[String]().to(sink).run()
+          val ref: Future[SinkRef[String]] = StreamRefs.sinkRef[String]().to(sink).run()
 
           // wrap the SinkRef in some domain message, such that the sender knows what source it is
-          val reply: MeasurementsSinkReady = MeasurementsSinkReady(nodeId, ref)
+          val reply: Future[MeasurementsSinkReady] = ref.map(MeasurementsSinkReady(nodeId, _))
 
           // reply to sender
-          sender() ! reply
+          reply pipeTo sender()
       }
```

Comment:
I do think that change is painful enough to not make it the final API. The API is now moved to `StreamRefs.` such that the good API can later be introduced on `Sink`/`Source` with a deprecation phase for the old ones ("playing nice")